proposals/PROPOSALS.md

```diff
@@ -0,0 +1,142 @@
+A HIP (*Haskell Foundation Initiative Proposal*) is a process for submitting a proposal for an initiative to be undertaken by the Haskell Foundation (HF). The main motivation of this document is to detail the primary mechanism through which the community may propose, discuss and implement initiatives to be carried out through the Haskell Foundation that will affect the Haskell ecosystem. For each HIP, all changes to the ecosystem go through a proposal process, which are is discussed by the HF Technical Track, the broader community, and any and all interested stakeholders. Only upon reaching a consensus are initiatives accepted to be merged and executed.
+
+The aim of the Haskell Improvement Process is to apply the openness and collaboration that have shaped Haskell's documentation and implementation to a process of evolving the HF and the broader Haskell ecosystem. This document captures our guidelines, commitments and expectations regarding this process.
+
+## Why Write a Proposal?
+
+HIPs are key to making the HF and its initiatives better for the good of everyone. If you decide to invest the time and effort of putting a proposal forward and see it through, your efforts and time will shape and improve the Haskell ecosystem, which means that your proposal may impact the life of a myriad of developers all over the world, including those on your own team. For many, this aspect alone can be quite worthwhile.
+
+However, it's important to note that seeing a proposal through to its conclusion is an involved task. On the one hand, it takes time to convince people that your suggestions are a worthwhile change for hundreds of thousands of developers to accept. Particularly given the sheer volume of developers that could be affected by a proposal, its acceptance is conservative and carefully thought through. Typically, this includes many rounds of discussion with HF leaders, Haskell library maintainers, and the broader community, several iterations on the design of the proposal, and some effort at prototyping the proposed change. Often, it takes weeks to months of discussion, re-design, and prototyping for a proposal to be accepted. It is therefore important to note that seeing a proposal through to its conclusion can be time-consuming and not all proposals may end up accepted, although they may teach us all something!
+
+If you’re motivated enough to go through this involved but rewarding process, go on with writing and keep on reading.
+
+## What's the process for submitting a HIP?
+
+There are four major steps in the HIP process:
+
+1. Initial informal discussion (2 weeks)
+2. Submission
+3. Formal presentation (up to 1 month)
+4. Formal evaluation (up to 2 months)
```

Comment:
After reading the rest of this document, either the order here is wrong, or I'm misunderstanding the process.

```suggestion
There are four major steps in the HIP process:

1. Initial informal discussion (2 weeks)
2. Submission
3. Formal evaluation and revisions
4. Formal presentation after acceptance
```

Reply:
I think this would be most accurate: 

```
1. Initial informal discussion 
2. Submission
3. Review
4. Acceptance
5. Formal presentation
```